public/localstorage-sync.html

```diff
@@ -37,8 +37,11 @@
                                     accounts[loginid] = token;
                                     tokens.push(token);
```

Comment:
Although it's not part of your changes, I was trying to understand the purpose of this line: tokens.push(token);. Could you or anyone else clarify what it's doing?

Reply:
true we actually dont need this here right, since we don't access this tokens variable, only the accounts one

Reply:
you're right . I removed it. maybe in the previous changes we needed it but then we forgot to remove it. thanks @hasan-deriv